docs/01-Introduction/03-retina-operator.md

```diff
@@ -0,0 +1,85 @@
+# Retina Operator
+
+## Overview
+
+The Retina Operator is a key control plane component responsible for orchestrating and managing the deployment, configuration, and lifecycle of Retina's functionality in a Kubernetes environment. Here is a detailed summary covering its role, function, configuration, and typical use cases:
+
+## Role and Responsibilities
+
+1. **Lifecycle Management**:
+    The operator manages the lifecycle of Retina custom resources, such as capture jobs and endpoint metadata. It watches for changes in the cluster and reconciles the desired state as specified by Kubernetes CRDs (Custom Resource Definitions) and user configurations.
+
+2. **Leader Election for High Availability**:
+    The operator uses Kubernetes leader election mechanisms to support high availability (HA) deployments. Only the elected leader performs management tasks, ensuring there are no conflicting operations in multi-replica setups.
+
+3. **CRD Management**:
+    It can install and manage the CRDs required for Retina's operation, ensuring that all necessary resources are available and up to date.
+
+4. **Configuration Propagation**:
+    The operator reads its configuration from a YAML file (default: `retina/operator-config.yaml`). It loads this configuration, applies defaults, and ensures all subcomponents (like telemetry and capture jobs) receive the correct parameters.
+
+5. **Telemetry and Logging**:
+    The operator can send telemetry data (if enabled) and manages logging for observability and debugging.
```

Comment:
To clarify, `manages logging for observability and debugging` is for operator itself, not anything else.